DecipherNow/gm-fabric-dashboard
# Fix: Toggle asc/desc Order in Service View

Relates to #1023 

## Proposed Changes

  - Enhanced `setSortByAttribute` instance method to handle toggling asc/desc order

## How to Test

  - Navigate to a JVM or GO service view
  - Sort instances by name, and then make the same selection. This should toggle the instances from asc to desc.
